src/pages/TributeNew.js

```diff
@@ -1,9 +1,99 @@
 
-import React from "react"
+import  React, {useState} from "react"
+import { Form, FormGroup, Input, Label, Button } from "reactstrap"
+import { useNavigate } from "react-router-dom"
+
+const TributeNew = ({createTribute}) => {
+    const [newTribute, setNewTribute] = useState ({
+        name: "",
+        age: "",
+        hobby: "",
+        image: "",
+        district: ""
+    })
+
+    const handleChange = (e) => {
+        setNewTribute({ ...newTribute, [e.target.name]: e.target.value })
+    }
+    const handleSubmit = () => {
+        createTribute(newTribute)
+        navigate("/tributeindex")
+    }
+    const navigate = useNavigate()
 
-const TributeNew = () => {
     return(
         <>
+        <h3>Add a Tribute</h3>
+        <Form>
+            <FormGroup>
+                <Label for="name">
+                Name
+                </Label>
+                <Input
+                id="name"
+                name="name"
+                placeholder="with a placeholder"
+                type="string"
+                onChange={handleChange}
+                value={newTribute.name}
+                />
+            </FormGroup>
+            <FormGroup>
+                <Label for="exampleage">
```

Comment:
for attribute of label tag should match the value on the id attribute of input tag